frontend/src/components/RecipeCardComponents/RecipeCard.js

```diff
@@ -4,7 +4,7 @@ import RecipeCardImage from "./RecipeCardImage";
 
 export default function RecipeCard({recipeCardDetails}) {
     return (
-        <CardWrapper to={`/recipe/${recipeCardDetails.id}`} key={recipeCardDetails.id}>
+        <CardWrapper to={`/recipe/${recipeCardDetails.id}/?from=recipes`} key={recipeCardDetails.id}>
```

Comment:
Just curious: Does the "from" mean that you are saving the previous route in the url? If so, why is that of importance?